src/out_hoi4/characters/out_character.cpp

```diff
@@ -7,6 +7,14 @@
 
 namespace
 {
+int GetGeneralSkill(const hoi4::General& general)
+{
+   return 1 + (general.attack + general.defense + general.logistics + general.planning - 4) / 3;
```

Comment:
Does this need to be clamped, or is it guaranteed to always produce a value in the right range? Likewise for admiral skills. And for the skills themselves, I guess.